### ISSUE TITLE
Compress-Archive does not produce the correct output as documented in the help

### ISSUE THREAD

**Issue by figueroadavid** [PowerShell/Microsoft.PowerShell.Archive#75]
This issue is present in Powershell 5.1, and 6.2.1 (verified)
The help for Compress-Archive shows that the output is `System.IO.FileInfo`.  However, when a zip file is successfully created, there is no output. 
Either the help should be updated, or the `System.IO.FileInfo` object should be produced.  The preference is that the object gets produced.  
_[update2] moving the -passthru piece to a separate issue_

# Steps to reproduce

```powershell
help Compress-Archive -full
PS C:\> compress-archive -path c:\demo -destinationpath c:\demo.zip
```

# Expected behavior

```
PS C:\> compress-archive -path c:\demo -destinationpath c:\demo.zip
Mode                LastWriteTime         Length Name
----                -------------         ------ ----
-a----       05/30/2019  10:56 PM           3686 demo.zip

```

# Actual behavior

```
PS C:\> compress-archive -path c:\demo -destinationpath c:\demo.zip
PS C:\>
```

# Environment data

```
Name                           Value
----                           -----
PSVersion                      6.2.1
PSEdition                      Core
GitCommitId                    6.2.1
OS                             Microsoft Windows 10.0.17134
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

**Comment by sdwheeler** [comment]
This was fixed in MicrosoftDocs/PowerShell-Docs#3696 on Feb 12. You need to run Update-Help -Force to get the new help files.

**Comment by sdwheeler** [comment]
This is a duplicate of issue #9.
